handlers/main.yml

```diff
@@ -3,7 +3,7 @@
   service:
     name: "{{ php_webserver_daemon }}"
     state: restarted
-  notify: restart php-fpm
+  notify: "{{ php_fpm_handler_action }} php-fpm"
   when: php_enable_webserver
 
 - name: restart php-fpm
```

Comment:
I'd rather just set the `state` on this task to a variable, instead of adding conditionals inside `notify` on all the tasks and also adding a new handler. Then the user can choose whether the state should be `restarted` or `reloaded` when the `restart php-fpm` handler is notified.

This new variable needs to also be documented in the README.

I would call the variable `php_fpm_handler_state` and default it to `restarted`.

Reply:
So we'll need to `notify: restart php-fpm` when just reload is wanted.
I think it should be confusing when reading the tasks.

Reply:
Why having 2 handlers should be worst?

Reply:
In the beginning I made just a variable, but just after I thought the same than @ncharlot : 1 restart handler which doesn't restart but reloads seems confusing to me.
That said, and you being owner of that repo, I can rewrite the branch to do what you asked, my main purpose is to get that feature merged into master :)

Reply:
Variable `php_fpm_handler_state` it is! :)